TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -273,14 +291,25 @@ public class IncompatibleModsPanel : UIPanel {
         /// </summary>
         /// <param name="mod">The <see cref="PluginInfo"/> associated with the mod that needs deleting.</param>
         /// <returns>Returns <c>true</c> if successfully deleted, otherwise <c>false</c>.</returns>
-        private bool DeleteLocalTMPE(PluginInfo mod) {
+        private bool DeleteLocalMod(PluginInfo mod) {
```

Comment:
is it worth it to even have that code in TM:PE?
this is convenience stuff and imo it should be in a separate mod.

Reply:
You mean that particular method or Panel/Extension in general?

Reply:
I mean the whole associated functionality. I have not checked what code is calling it though.

Reply:
all is custom, reads incompatible mods from txt file (assembly resource) or that hashset above (if assembly name is known - required for local mods) 

I agree that might be useful for [Compatibility Report](https://steamcommunity.com/sharedfiles/filedetails/?id=2633433869)

Reply:
so .. yeah .. as long as we don't have a "compatibility report" mod that is the goto for the whole community it makes sense to have some functionality in TM:PE that checks for imcompatible mods and displays some popup.
What I mean is the code to immediately delete the incompatible mod. I would not put that into TM:PE. but i won't mind if it stays too :P

Reply:
anyway ... MERGE IT already!! :D